packages/Snackbar/README.md

```diff
@@ -154,6 +154,10 @@ To add some action to the Snackbar, just add options about the action to the sho
 ![Feb-25-2020 00-16-47](https://user-images.githubusercontent.com/17980230/75164429-265a0000-5764-11ea-9c6f-12bf362dc32b.gif)
 
 
+<<<<<<< HEAD
```

Comment:
If some conflict occurred when rebasing, you should check the conflict and fix it before add and continue rebasing.
Please reset to the previouse commit and rebase again.

Reply:
sorry, I missed conflict... then your advice means rebase recent two commit for combining to one commit?
I confuse because my work is already contained the https://github.com/dooboolab/dooboo-ui/commit/1a615818ea2995248969a7447612605f51970e2d. Didn't you already rebase it?

Reply:
You're right. My suggestion assumes that you know the commit number before rebase. And this is non-essential and rather an inconvenient way. 
I want just that this commit not included at the master branch. So, could you fix this and commit using `--amend` option and force push again?